pkg/controller/vitessshardreplication/vitessshardreplication_controller.go

```diff
@@ -226,3 +216,29 @@ func (r *ReconcileVitessShard) Reconcile(request reconcile.Request) (reconcile.R
 	reconcileCount.WithLabelValues(metricLabels(vts, err)...).Inc()
 	return result, err
 }
+
+func (r *ReconcileVitessShard) initReplication(ctx context.Context, vts *planetscalev2.VitessShard, wr *wrangler.Wrangler) (reconcile.Result, error){
+	resultBuilder := &results.Builder{}
+
+	// If we have configured the operator to not initialize replication, bail.
+	if !vts.Spec.VitessShardTemplate.Replication.InitializeMaster {
```

Comment:
VitessShardTemplate is an anonymous field so we usually omit it for clarity.

```suggestion
	if !*vts.Spec.Replication.InitializeMaster {
```

Reply:
Done.

Reply:
Done.